src/views/record-browser/table-data/table-data.html

```diff
@@ -1,15 +1,23 @@
 <table>
 	<thead>
 	<tr>
+		<th></th>
```

Comment:
This table heading needs `class="actions"` to match the table data it describes. If a table has no records, the width of this column is undefined.

![image](https://cloud.githubusercontent.com/assets/13726377/16359313/2ed00d9c-3ae4-11e6-8301-eb84aaea42a1.png)